commands/faas.go

```diff
@@ -31,6 +31,13 @@ var (
 	language     string
 )
 
+// TODO: remove this workaround once these vars are no longer global
```

Comment:
@ericstoekl can we do this from within the tests instead? Would it work because I don't like this here, sorry.

Reply:
Yes, the usage of it is only in the test files anyway, so it should be in `test/utils.go`

Reply:
My intent was to put in utils originally, but I could not access the variables from there. Dont know much about go, but seemed like a package scoping issue. Didnt want to get too invasive. Having to refactor or reflect.